### DISCUSSION TITLE
Snapshot Interval - Is it seconds? minutes?

### DISCUSSION THREAD

**Discussion by trentnbauer** [tsightler/ring-mqtt#718] [Support Q&A] [answered]
Hi all,

Silly question, sorry.

Is the snapshot internal in seconds?

**Comment by tsightler** [comment, marked as answer]
Seconds.  I agree it could be more clear, but the documentation does state that the defaults are 30 seconds for high-power cameras and 600 seconds for low-power cameras so that's at least a pretty decent hint that it is in seconds.  I need to find time to document all device level settings, I have a section in the wiki for it, but not much time to work on it.
